File: python/fz.py
def get_fizz_buzz_list():
    res = []
    for i in range(1, 101):
        s = ''
        if i % 3 == 0:
            s += "Fizz"
        if i % 5 == 0:
            s += "Buzz"
        if i % 3 != 0 and i % 5 != 0:
            s += str(i)
        res.append(s)
    return res

print(get_fizz_buzz_list())

File: perl/fz.pl
use strict;
use warnings;

sub getfizzbuzzlist() {
  my $i;
  my @list = ();
  foreach $i (1..100) {
    if ($i % 3 == 0) {
      push(@list, "Fizz");
    }
    if($i % 5 == 0) {
      push(@list, "Buzz");
    }
    push(@list, $i);
  }
  return(@list);
}

# test
# my @fizzbuzzlist = &getfizzbuzzlist();
# print(@fizzbuzzlist)
# -> 12Fizz34Buzz5Fizz678Fizz9Buzz10...
